### PR TITLE
Make value names a bit more consistent where we use them so we can be…

### PR DESCRIPTION
**Pull request by ScottDowne**
… sure they are cleared on input, and handle the correct type vs rawType from stripe errors.

### CODE REVIEW

File: mixins/form.jsx [hunk 1]
```diff
@@ -264,13 +265,13 @@ module.exports = {
       exp_year: submitProps.expYear
     }, function(status, response) {
       if (response.error) {
-        error(response.error);
+        error(response.error.code, response.error.type);
```
Comment (ScottDowne) [inline mixins/form.jsx]: This is the main fix. createToken errors return error.type, and stripe.charges.create return error.rawType.

Both are the values we want.

File: mixins/form.jsx [hunk 2]
```diff
@@ -264,13 +265,13 @@ module.exports = {
...
       } else {
         submitProps.cardNumber = "";
         submitProps.stripeToken = response.id;
         submit("/api/stripe", submitProps, function(result) {
           if (result.error) {
-            error(result.error);
+            error(result.error.code, result.error.rawType);
```
Comment (ScottDowne) [inline mixins/form.jsx]: See above comment.

File: mixins/form.jsx [hunk 3]
```diff
@@ -25,10 +25,10 @@ module.exports = {
       errors: {
         creditCardInfo: {
           page: 0,
-          number: "",
+          cardNumber: "",
           cvc: "",
-          monthExp: "",
-          yearExp: ""
+          expMonth: "",
+          expYear: ""
```
Comment (ScottDowne) [inline mixins/form.jsx]: Just making these the same as other similar values we use in other places.

**Comment by ScottDowne** [comment]
R?

**Comment by alicoding** [comment]
r+ from me

**Pull request merged**